sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/CosmosOperations.java

```diff
@@ -123,6 +125,28 @@ public interface CosmosOperations {
      */
     <T> T insert(T objectToSave, PartitionKey partitionKey);
 
+    /**
+     * patches item
+     * @param id must not be {@literal null}
+     * @param partitionKey must not be {@literal null}
+     * @param domainType must not be {@literal null}
```

Comment:
would be nice to add a description on what `domainType` is

Reply:
agreed.